peps/pep-0771.rst

```diff
@@ -425,16 +387,16 @@ explicitly specify a frontend, e.g.:
 
     $ pip install napari[pyside6]
 
-Supporting minimal installations while not always removing default extras
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Supporting extras that should not remove default extras
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-An additional case we consider here is where a package maintainer wants to support
-minimal installations without any extras, but also wants to support having users
-specify additional extras without removing the default one. Essentially, they
-would want:
+An additional case we consider here is where a package maintainer wants to
+support the ability for users to opt-in to non-default extras, without removing
+default extras. Essentially, they might want:
 
-* ``package[minimal]`` to give an installation without any extras
+* ``package[]`` to give an installation without any extras
 * ``package`` to install recommended dependencies (in a ``recommended`` extras)
+* ``package[alternative]`` to not install default extras, but to install an alternative set of optional dependencies (in an ``alternative`` extras)
```

Comment:
Do we anticipate `alternative` and `additional` to be commonly used? If so, it may make sense somewhere in the PEP, even an appendix to provide how that differs from `dev`, `testing`, `docs`, etc. today.

Reply:
No, ``alternative`` and ``additional`` are not names that I expect people to use - they are just example names to distinguish between extras that add dependencies and ones that replace the default ones.